comps/comps-katello-client-rhel7.xml

```diff
@@ -18,10 +18,7 @@
       <packagereq type="default">katello-host-tools-tracer</packagereq>
       <packagereq type="default">katello-host-tools</packagereq>
       <packagereq type="default">pulp-rpm-handlers</packagereq>
```

Comment:
In el7 we are using the 'new' katello agent code which has the pulp libs packaged with it

I *think* we can get rid of: pulp-rpm-handlers, python-pulp-agent-lib, python-pulp-common, python-pulp-rpm-common, python-qpid-common, python-saslwrapper, saslwrapper,  python-isodate

Reply:
I show the following and will cleanup based on that:

```
package pulp-rpm-handlers is not installed
package python-pulp-agent-lib is not installed
python-pulp-common-2.16.2-1.el7.noarch
python-pulp-rpm-common-2.16.2-1.el7.noarch
package python-qpid-common is not installed
python-saslwrapper-0.22-5.el7.x86_64
saslwrapper-0.22-5.el7.x86_64
python-isodate-0.5.4-8.el7.noarch
```

Reply:
Sorry, I take that back. Looking at things:

```
Installing:
 katello-agent                                                        noarch                               3.2.1-4.el7                                          katello-client                                19 k
Installing for dependencies:
 gofer                                                                noarch                               2.12.1-1.el7                                         jortel-gofer                                  51 k
 katello-host-tools                                                   noarch                               3.2.1-4.el7                                          katello-client                                28 k
 katello-host-tools-fact-plugin                                       noarch                               3.2.1-4.el7                                          katello-client                                12 k
 libnl                                                                x86_64                               1.1.4-3.el7                                          base                                         128 k
 pulp-rpm-handlers                                                    noarch                               2.16.2-1.el7                                         pulp                                          80 k
 python-dateutil                                                      noarch                               1.5-7.el7                                            base                                          85 k
 python-dmidecode                                                     x86_64                               3.12.2-2.el7                                         base                                          83 k
 python-ethtool                                                       x86_64                               0.8-5.el7                                            base                                          33 k
 python-gofer                                                         noarch                               2.12.1-1.el7                                         jortel-gofer                                 193 k
 python-gofer-proton                                                  noarch                               2.12.1-1.el7                                         jortel-gofer                                  57 k
 python-inotify                                                       noarch                               0.9.4-4.el7                                          base                                          49 k
 python-isodate                                                       noarch                               0.5.4-8.el7                                          base                                          54 k
 python-pulp-agent-lib                                                noarch                               2.16.2-1.el7                                         pulp                                          92 k
 python-pulp-common                                                   noarch                               2.16.2-1.el7                                         pulp                                         124 k
 python-pulp-rpm-common                                               noarch                               2.16.2-1.el7                                         pulp                                          73 k
 python-six                                                           noarch                               1.9.0-2.el7                                          base                                          29 k
 python2-qpid-proton                                                  x86_64                               0.22.0-1.el7                                         epel                                         219 k
 qpid-proton-c                                                        x86_64                               0.22.0-1.el7                                         epel                                         183 k
 subscription-manager                                                 x86_64                               1.20.11-1.el7.centos                                 base                                         993 k
 subscription-manager-rhsm                                            x86_64                               1.20.11-1.el7.centos                                 base                                         285 k
 subscription-manager-rhsm-certificates                               x86_64                               1.20.11-1.el7.centos                                 base                                         195 k
 usermode                                                             x86_64                               1.111-5.el7                                          base                                         193 k

Transaction Summary
```

I think we still need all of those.

Reply:
It looks like the offender is the katello-agent package itself. This needs to be updated. I'll test the theory and open a PR

Reply:
what version of the agent are you installing? those deps shouldn't be brought in 3.3.0-1

Reply:
Ahhh, good catch:

```
===================================================================================================================================================================================================================
 Package                                                             Arch                                Version                                            Repository                                        Size
===================================================================================================================================================================================================================
Installing:
 katello-agent                                                       noarch                              3.3.0-1.el7                                        katello-client-koji                               37 k
Installing for dependencies:
 gofer                                                               noarch                              2.12.1-1.el7                                       jortel-gofer                                      51 k
 katello-host-tools                                                  noarch                              3.3.0-1.el7                                        katello-client-koji                               28 k
 katello-host-tools-fact-plugin                                      noarch                              3.3.0-1.el7                                        katello-client-koji                               12 k
 libnl                                                               x86_64                              1.1.4-3.el7                                        base                                             128 k
 python-dateutil                                                     noarch                              1.5-7.el7                                          base                                              85 k
 python-dmidecode                                                    x86_64                              3.12.2-2.el7                                       base                                              83 k
 python-ethtool                                                      x86_64                              0.8-5.el7                                          base                                              33 k
 python-gofer                                                        noarch                              2.12.1-1.el7                                       jortel-gofer                                     193 k
 python-gofer-proton                                                 noarch                              2.12.1-1.el7                                       jortel-gofer                                      57 k
 python-inotify                                                      noarch                              0.9.4-4.el7                                        base                                              49 k
 python-six                                                          noarch                              1.9.0-2.el7                                        base                                              29 k
 python2-qpid-proton                                                 x86_64                              0.22.0-1.el7                                       epel                                             219 k
 qpid-proton-c                                                       x86_64                              0.22.0-1.el7                                       epel                                             183 k
 subscription-manager                                                x86_64                              1.20.11-1.el7.centos                               base                                             993 k
 subscription-manager-rhsm                                           x86_64                              1.20.11-1.el7.centos                               base                                             285 k
 subscription-manager-rhsm-certificates                              x86_64                              1.20.11-1.el7.centos                               base                                             195 k
 usermode                                                            x86_64                              1.111-5.el7                                        base                                             193 k

Transaction Summary
===================================================================================================================================================================================================================
Install  1 Package (+17 Dependent packages)
```